### ISSUE TITLE
configurable reporting interval vs/ fixed 5-min

### ISSUE THREAD

**Issue by boucadair** [ietf-wg-alto/draft-ietf-alto-oam-yang#19]
The current stat module uses a fix interval to report the stats. I would prefer having a configurable interval.

**Comment by billwuqin** [comment]
Agree, I am wondering how  time-window-size is related to this fixed 5 min time interval.
Check the latest version of YANG data model, I see three places use fixed 5 min:
       leaf num-total-last-req {
         type yang:counter64;
         config false;
         description
           "The total number of ALTO requests received by the server
            within the last 5 minutes.";
       }
       leaf num-total-last-succ {
         type yang:counter64;
         config false;
         description
           "The total number of successful responses sent by the ALTO
            server within the last 5 minutes.";
       }
       leaf num-total-last-fail {
         type yang:counter64;
         config false;
         description
           "The total number of failed responses sent by the ALTO
            server within the last 5 minutes.";
       }

**Comment by fno2010** [comment]
I also agree. We have changed the fixed reporting interval to a configured parameter:

https://github.com/ietf-wg-alto/draft-ietf-alto-oam-yang/blob/5dea617f7e681a51666c173adab7c575bc1cbb23/yang/ietf-alto-stats.yang#L278-L285
